scripts/lib/CIME/case_run.py

```diff
@@ -120,7 +121,7 @@ def _run_model_impl(case, lid):
                         restore_from_archive(case)
 
                         orig_cont = case.get_value("CONTINUE_RUN")
-                        if not orig_cont:
+                        if not (orig_cont or skip_pnl):
```

Comment:
Maybe @jedwards4b can help me here. I don't know if skipping create_namelists here is safe if orig_cont is False. run_model can be invoked several times for certain tests and in other situations.